virttest/test_setup.py

```diff
@@ -930,7 +930,9 @@ def __init__(self, driver=None, driver_option=None, host_set_flag=None,
                 if "allow_unsafe_assigned_interrupts" in i:
                     self.auai_path = i
         if self.setup:
-            self.sr_iov_setup()
+            if not self.sr_iov_setup():
+                msg = "SR-IOV setup on host failed"
+                logging.error(msg)
```

Comment:
@bssrikanth you can use either 
`
from avocado.core import exceptions
raise exceptions.TestError(msg)`
or
`raise exceptions.TestSkipError(msg)`